src/django_nh3/templatetags/nh3_tags.py

```diff
@@ -0,0 +1,25 @@
+import nh3
+from django import template
+from django.utils.safestring import SafeText, mark_safe
+
+from django_nh3.utils import get_nh3_default_options
+
+register = template.Library()
+
+
+@register.filter(name="nh3")
+def nh3_value(value: str | None, tags: str | None = None) -> SafeText:
```

Comment:
Can you add a docstring on this please? Nothing complex, just something basic that descibes the fact it takes a value and will run clean on it & return a safe value to include in the template.

Reply:
Just added! Not sure if there's a specific format you prefer so I can add to it if you have a preference. Cheers!

Reply:
Perfect - thank you!